### PR TITLE
エラー画面が表示されるようにクラスを作成しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/error")
+public class ErrorController {
+	@GetMapping("/4xx")
+	public String error4xx() {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+		return "/error/4xx";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ErrorController.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package jp.co.example.ecommerce_a.common;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java:1]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.PackageNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name 'jp.co.example.ecommerce_a.common' must match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'.

File: src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.springframework.boot.web.server.ErrorPage;
+import org.springframework.boot.web.server.WebServerFactoryCustomizer;
+import org.springframework.boot.web.servlet.server.ConfigurableServletWebServerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java:9]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+public class CustomConfigration implements WebServerFactoryCustomizer<ConfigurableServletWebServerFactory> {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java:10]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 108).

File: src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+    @Override
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    public void customize(ConfigurableServletWebServerFactory factory) {
+        factory.addErrorPages(new ErrorPage(HttpStatus.NOT_FOUND, "/error/4xx"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    }
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/CustomConfigration.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 10]
```diff
@@ -25,7 +25,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		// アクセスの制限に関する設定
 		http.authorizeHttpRequests(ahr -> ahr
 				.requestMatchers("/", "/registerUser/**", "/login/**", "/showItemDetail/**",
-						"/showShoppingCart/**", "/addShoppingCart/**", "/deleteShoppingCart/**","/narrowingDown","/searchAndSort")
+						"/showShoppingCart/**", "/addShoppingCart/**", "/deleteShoppingCart/**","/narrowingDown","/searchAndSort","/error/**")
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 166).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.

**Review by yamada-kensyu — APPROVED** [review]
承認します！

**Pull request merged**